astro/src/content/docs/release-notes/index.mdx

```diff
@@ -38,6 +38,21 @@ import IssueResolvedVia from 'src/components/docs/release-notes/ReleaseNoteIssue
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.53.3' releaseDate='October 22, 2024'/>
+
+### Fixed
+<Issue issue="2764">
+The CSRF token for federated login added in version `1.47.0` is not applied when all identity providers configured for an application use managed domains, and the OAuth authorize request for the application includes an `idp_hint` parameter.
+</Issue>
+
+<Issue issue="2893,2472">
```

Comment:
This is ok if we are truly solving two issues. But it may be best to just close on GH issue as a duplicate, or solved via, and then let that be the trail instead of listing both here? 

Either way is fine really. But if it was a true duplicate, I would say we would want to close one GH issue as a dup, and then just list the one here that we are resolving. 

We could do that here since we did close the one GH issue as resolved via.

Reply:
These issues are kind of distinct and not duplicate. One is for the old behavior, one is for the new behavior. Technically the last release invalidated #2472 by reversing its behavior and creating #2893. I will remove it here and close it with a note.